marshaling.go

```diff
@@ -132,6 +132,30 @@ type WorksheetConverter interface {
 var worksheetConverterType = reflect.TypeOf((*WorksheetConverter)(nil)).Elem()
 
 func (ws *Worksheet) StructScan(dest interface{}) error {
+	// mapping function returns ws field, ignore/skip, error
+	// this function uses tags
+	f := func(refWs *Worksheet, currentField reflect.StructField) (*Field, bool, error) {
```

Comment:
refWs will always be available at the call point to ws.StructScanWithNameFunc, so not strictly necessary to pass it in since it can be closed on. having it as a parameter makes the use more explicit and perhaps allows the function to come in from a different context, but could take out.